modules/end-user-guide/partials/proc_building-an-image-for-intellij-idea-community-or-pycharm-community.adoc

```diff
@@ -0,0 +1,99 @@
+[id="building-an-image-for-intellij-idea-community-or-pycharm-community_{context}"]
+= Building an image for IntelliJ IDEA Community or PyCharm Community
+
+This procedure describes how to build an image for IntelliJ IDEA Community or PyCharm Community.
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has an installation of following tools:
+
+** link:https://www.docker.com/[Docker] version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit]
+
+** link:https://git-scm.com[Git]
+
+** link:https://www.gnu.org/software/libc/manual/html_node/Getopt.html[GNU getopt]
+
+** link:https://www.gnu.org/software/wget/[GNU wget]
+
+**  link:https://openjdk.java.net/projects/jdk/11/[Java Development Kit (JDK) version 11]
+
+** link:https://stedolan.github.io/jq/[jq]
+
+
+.Procedure
+
+. Get a local copy of the link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains Projector Editor Images repository].
++
+----
+$ git clone https://github.com/che-incubator/jetbrains-editor-images
+$ cd jetbrains-editor-images
+----
+
+. Run the build script and select the IDE package and package version:
++
+----
+$ ./projector.sh build
+[info] Select the IDE package to build (default is 'IntelliJ IDEA Community'):
+       1) IntelliJ IDEA Community
+       2) PyCharm Community
+[info] Select the IDE package version to build (default is '2020.3.3'):
+       1) 2020.3.3
+       2) 2020.3.2
+       3) 2020.3.1
+----
+
+. To test the image, run it locally and navigate to `++http://localhost:8887++` to access the IDE.
```

Comment:
IMB Style, p. 314:
_navigate verb
Use only when a more direct verb, such as "click", "select", "browse", or "go to", is not applicable. Use only to provide direction to a location (for example, "Navigate to the directory that contains the files that you want to edit")._
imho `go to` is easier to understand if you just mean opening a website.

Reply:
```suggestion
. To test the image, run it locally and go to `++http://localhost:8887++` to access the IDE.
```